src/tools/idlc/src/descriptor_type_meta.c

```diff
@@ -1752,33 +1752,48 @@ generate_type_meta_ser (
   const idl_node_t *node,
   idl_typeinfo_typemap_t *result)
 {
```

Comment:
Assuming `result` points to uninitialised memory (perfectly reasonable for an out parameter) ...